ams/ams-server/src/test/java/com/netease/arctic/ams/server/optimize/TestMajorOptimizePlan.java

```diff
@@ -112,7 +112,7 @@ public void testKeyedTableFullOptimize() throws IOException {
         .collect(Collectors.toList()));
 
     testKeyedTable.updateProperties()
-        .set(TableProperties.FULL_OPTIMIZE_TRIGGER_DUPLICATE_SIZE_BYTES_THRESHOLD, "0")
+        .set(TableProperties.SELF_OPTIMIZING_MAJOR_TRIGGER_DUPLICATE_RATIO, "0.00000001")
```

Comment:
Why does the ratio change to 0.00000001?

Reply:
We need a small ratio to trigger `Optimizing`.  
However, `0` may be better here, I will fix it.